docs/README.md

```diff
@@ -0,0 +1,288 @@
+Copyright 2023 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+# Performance PSI Tracker
+
+<img src="./imgs/psi_tracker_screenshot.webp" alt="Screenshot of the Performance PSI Tracker" width="60%" >
+
+* [About the Performance PSI Tracker](#about-the-performance-psi-tracker)
+* [Setting up the Tracker](#setting-up-the-tracker)
+    - [Getting a PSI API Key](#getting-a-psi-api-key)
+    - [Configuring the Tracker](#configuring-the-tracker)
+* [Using the Performance PSI Tracker](#using-the-performance-psi-tracker)
+* [Visualizing your Metrics](#visualizing-your-metrics)
```

Comment:
How would you feel adding indications to the two options (ie: Sheets, Looker Studio) here? This would give a heads up that there are multiple ways to visualize the data.